openfisca_us/variables/gov/states/in/tax/income/deductions/in_rent.py

```diff
@@ -1,10 +1,11 @@
 from openfisca_us.model_api import *
 
 
-class in_deductions(Variable):
+class in_rent(Variable):
```

Comment:
Good to be precise, but since many states have rental credits/deductions and taxsim unifies them, let's use our existing `rent` variable here (this will also save space). Could you add a comment in the `in_renters_deduction` variable indicating the restrictions (principal place of residence subject to IN property tax)?